ArcGISRuntimeSDKQt_CppSamples/Layers/AddClusteringFeatureReductionToAPointFeatureLayer/AddClusteringFeatureReductionToAPointFeatureLayer.qml

```diff
@@ -0,0 +1,127 @@
+// [WriteFile Name=AddClusteringFeatureReductionToAPointFeatureLayer, Category=Layers]
+// [Legal]
+// Copyright 2023 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick
+import QtQuick.Controls
+import Esri.Samples
+import Esri.ArcGISRuntime.Toolkit
+import QtQuick.Layouts
+
+Item {
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: parent
+
+        Component.onCompleted: {
+            // Set and keep the focus on MapView to enable keyboard navigation
+            forceActiveFocus();
+        }
+    }
+
+    // Declare the C++ instance which creates the map etc. and supply the view
+    AddClusteringFeatureReductionToAPointFeatureLayerSample {
+        id: sample
+        mapView: view
+    }
+
+    PopupStackView {
```

Comment:
The popup is empty, I don't understand why

Reply:
Before clustering is enabled, popup view displays correctly the identified layer. And with clustering, layer content's name is correct. It looks like the identification and the popup view are correct.